src/dwarf_parser.h

```diff
@@ -43,6 +41,17 @@ class Dwarf {
   void resolve_fields(std::shared_ptr<Struct> str, lldb::SBType type);
   std::optional<Bitfield> resolve_bitfield(lldb::SBTypeMember field);
 
+  // Initialize/Terminate liblldb globally with RAII
+  class InstanceCounter {
+  private:
+    static std::atomic<size_t> count;
+
+  public:
+    InstanceCounter();
+    virtual ~InstanceCounter();
```

Comment:
Good point, it doesn't need to be, as this class is not inherited from.
I've remove the `virtual`, but marked the `InstanceCounter` class as `final` to prevent any inheritance.